package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "preact-realworld-example-app",
-  "private": "true",
-  "homepage": "https://github.com/mutoe/preact-realworld-example-app#readme",
+  "version": "0.1.2",
+  "private": true,
```

Comment:
I'd say there's still value in versioning a web app, but meta data like the home page? Not really sure about when that would be useful?

Reply:
If a user wants to view its online repository after opening the source code, this will help him, I think

Reply:
If the user closes the source then they'll already know of the upstream?

Reply:
It doesn’t matter, I don’t mind these ;p